### PR TITLE
refactor: code

### PR DESCRIPTION
**Pull request by alexander-akait**
- [ ] This is a **bugfix**
- [x] This is a **feature**
- [x] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

Yes

### Motivation / Use-Case

union `open` and `openPage`.

### Breaking Changes

Yes, `openPage` was removed

### Additional Info

```ts
type OpenTarget = string | string[];
// i.e. `'google-chrome'` or `['google-chrome', '--incognito']`
type OpenApp = string | string[];
type OpenObject = { target?: OpenTarget, app?: OpenApp };
type Open = boolean | OpenTarget | OpenObject | (OpenTarget | OpenObject)[] ;
```

### CODE REVIEW

**Comment by alexander-akait** [comment]
Also fixed cases when developer uses `['google-chrome', '--incognito']`

**Comment by alexander-akait** [comment]
Hm, maybe we don't need `--open` anymore for CLI, maybe we can use `--open-target`, but let's keep for compatibility right now, we will think about this for the next major release

File: lib/Server.js [hunk 1]
```diff
@@ -719,7 +719,7 @@ class Server {
     if (this.options.open || this.options.openPage) {
```
Comment (snitin315) [inline lib/Server.js:719]:
```suggestion
    if (this.options.open) {
```

**Comment by alexander-akait** [comment]
I thought and decided that we will keep `--open` cli option for compatibility purpose, we will remove it in the next major release (we already do a lot of breaking changes and I think we will do it more), so this release will be painful for some developers, let's reduce it where possible

**Comment by snitin315** [comment]
Makes sense 👍

**Pull request merged**